src/components/Kv/KvResponsiveImage2.vue

```diff
@@ -0,0 +1,136 @@
+<template>
```

Comment:
Goal of this component is to use the same api as KvResponsiveImage but output a picture element with <source> and srcset tags in order to take advantage of the look-ahead browser rendering of images. 
With the exististing KvResponsiveImage, images fetching doesn't start until after domcontentloaded because we need to wait for JS to query the screenwidth and density before deciding setting the image src. 

Browsers that don't understand the picture element (IE11) use the same logic that we had before. Browsers that do understand picture get significantly earlier image downloads.

Reply:
Nice! Thanks for the explanation @ryan-ludwig